lib/reek/cli/command.rb

```diff
@@ -1,16 +0,0 @@
-module Reek
-  module CLI
-    #
-    # Base class for all commands
-    #
-    class Command
```

Comment:
>>  a), since you're jumping between module and class?

Yes. True. I still find it better readable as it is right now, but that might just be me.

>> Does b) mean the Command pattern violates Liskov?
 
Let me contrast the command pattern:

>> In object-oriented programming, the command pattern is a behavioral design pattern in which an object is used to encapsulate all information needed to perform an action or trigger an event at a later time.

with the substitution principle:

>> Substitutability is a principle in object-oriented programming. It states that, in a computer program, if S is a subtype of T, then objects of type T may be replaced with objects of type S (i.e., objects of type S may substitute objects of type T) without altering any of the desirable properties of that program (correctness, task performed, etc.). 

I think it depends on your point of view. From a "command pattern" point of view where I just execute "something" it doesn't. My angle was coming from object orientated thinking and the fact that those 2 commands are completely different. In fact, they couldn't be more different. 

But while writing this down I realize that i'm probably yak-shaving right now. This discussion would make more sense if we had dozens of different commands. 
I'll rewire the commands as separate classes.